mappings/net/minecraft/loot/OneTwentyLootTables.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/unmapped/C_hcsgzkdg net/minecraft/loot/OneTwentyLootTables
```

Comment:
That's not exactly all "OneTwenty" loot tables, it's just for archaeology.

```suggestion
CLASS net/minecraft/unmapped/C_hcsgzkdg net/minecraft/loot/OneTwentyArchaeologyLootTables
```

Reply:
alright, though I wonder how Mojang came up with the mojmap name then (It's `UpdateOneTwentyBuiltInLootTables`).